web/src/pages/app/functions/mods/DependecePanel/AddDepenceModal/index.tsx

```diff
@@ -60,14 +67,6 @@ const AddDepenceModal = forwardRef((_, ref) => {
     setList(newList);
   });
 
-  useImperativeHandle(ref, () => ({
-    edit: (item: TPackage) => {
-      // setItem(item);
-      setIsEdit(true);
-      onOpen();
-    },
-  }));
-
   const search = useCallback(
     loadsh.debounce((val: string) => {
```

Comment:
spelling error: loadsh  -> lodash ， 

and `import { debounce } from 'lodash'` should be used ，instead of `import lodash from 'lodash'`

Reply:
<img width="714" alt="image" src="https://user-images.githubusercontent.com/27497945/209649821-5cbe3a23-aeb4-42ea-a4ad-341c6f66bf02.png">